Tasks/ANT/ant.ps1

```diff
@@ -108,9 +108,15 @@ if ($jdkPath)
 }
 
 $buildRootPath = Split-Path $antBuildFile -Parent
-$reportDirectoryName = [guid]::NewGuid()
+$reportDirectoryName = "ReportDirectory"
 $reportDirectory = Join-Path $buildRootPath $reportDirectoryName
 
+if(Test-Path $reportDirectory)
```

Comment:
This wont be a problem in multiple maven tasks as prawal said. I tested this case while doing for ts implementation